docs/walk-through/authentication.md

```diff
@@ -100,6 +101,55 @@ class MyTokenGenerator(TokenGenerator):
 global_config.host = "https://my-argo-server.com"
 global_config.token = MyTokenGenerator
 
+
+# the workflow automatically creates a workflow service, which uses the global config
+# host and token generator for authentication
+with Workflow(
+    generate_name="test-",
+    entrypoint="c",
+) as w:
+    Container(name="c", image="alpine:3.13", command=["sh", "-c", "echo hello world"])
+
+w.create()
+```
+
+##### Generating a K8s token via `TokenGenerator`
+
+```python
+import base64
+import errno
+import os
+from typing import Optional
+
+from kubernetes import client, config
```

Comment:
This would require users to pip install kubernetes right? Could we make a note of that

Reply:
Sure thing!